userspace/engine/falco_utils.cpp

```diff
@@ -151,6 +151,13 @@ std::string calculate_file_sha256sum(const std::string& filename)
 }
 #endif
 
+std::string sanitize_metric_name(const std::string& name)
+{
+	std::string sanitized_name = name;
+	RE2::GlobalReplace(&sanitized_name, "\\s+", "_");
```

Comment:
Are we sure that only white spaces are the problematic characters that can be found in a metric name ?

Reply:
i read your comment after I pushed lol, repushed just now.